inputmodule/inputModule.py

```diff
@@ -0,0 +1,20 @@
+def input_loop():
+    while True:
+        prompt = raw_input(">").split(" ")
+        if prompt == "help":
+            print("Type 'set [var] [value]' to set a variable to a value\nType 'get [var]' to get the value stored in the variable output as a string\navailable variables are yaw, pitch, output, kernel")
+        if prompt == "exit":
+            return
+        if prompt == "get":
+            get()
+        if prompt == "set":
+            set()
+input_loop()
```

Comment:
Get rid of this line, or the function will run each time the script is imported.